frontends/web/src/api/bitbox02.ts

```diff
@@ -41,6 +41,19 @@ export const getDeviceInfo = (
         });
 };
 
+export interface VersionInfo {
```

Comment:
Since you touch this, could you change to `type = `? We discussed a while ago, main reason that there is no accidental redeclaration / extending.

https://dev.to/reyronald/typescript-types-or-interfaces-for-react-component-props-1408#redeclaration

Reply:
done :+1: